src/components/Dashboard/Form/FieldWrapper/SelectOptions.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const Opts = props => {
+  return (
+    <select name={props.name} onChange={props.onChange}>
+      {props.options.map(x => {
+        return (
+          <option key={x.key} value={x.value}>
+            {x.text}
+          </option>
```

Comment:
Probably `return` is not necessary
```suggestion
          {props.options.map(x => (
           <option key={x.key} value={x.value}>
             {x.text}
           </option>
          )}
```

Reply:
Or destructing x object
```suggestion
{props.options.map(x => {
const {key, value, text} = x;
return (
 <option key={key} value={value}>{text}</option>
)
})